packages/comms/src/services/wsDFU.ts

```diff
@@ -493,6 +508,8 @@ export namespace WsDfu {
         Persistent: boolean;
         IsProtected: boolean;
         KeyType: string;
+        NumOfSubfiles: number;
+        Accessed: string;
```

Comment:
@GordonSmith Not really relevant to this pr, but I was wondering if TS had something like a date type? I noticed there were a handful of members in a few interfaces in here that, like this one, looked like they were most likely dates. Or maybe these date-like strings don't necessarily always conform to the same format when they come in? Might not be useful or worth changing.

Reply:
They are somewhat consistent, but do not match the JS Date serialization format...

Typically:
* Date:  2021-05-21
* Time:  13:33:22.123
* Date Time:  2021-05-21T13:33:22.123Z  (ISO Date Time I think)